internal/datanode/importv2/util.go

```diff
@@ -105,7 +105,7 @@ func NewSyncTask(ctx context.Context,
 
 func NewImportSegmentInfo(syncTask syncmgr.Task, metaCaches map[string]metacache.MetaCache) (*datapb.ImportSegmentInfo, error) {
 	segmentID := syncTask.SegmentID()
-	insertBinlogs, statsBinlog, deltaLog := syncTask.(*syncmgr.SyncTask).Binlogs()
+	insertBinlogs, statsBinlog, bm25Log, deltaLog := syncTask.(*syncmgr.SyncTask).Binlogs()
```

Comment:
incorrect order, maybe we could use a new struct to prevent incorrect order, like:
strcut SyncTaskBinlogs {
  insertBinlogs *FieldBinlog
  statsBinlog *FieldBinlog
  deltaLog *FieldBinlog
  bm25Log *FieldBinlog
}